src/wallet/walletdb.cpp

```diff
@@ -441,10 +441,10 @@ ReadKeyValue(CWallet* pwallet, CDataStream& ssKey, CDataStream& ssValue,
             // Extract some CHDChain info from this metadata if it has any
             if (keyMeta.nVersion >= CKeyMetadata::VERSION_WITH_HDDATA && !keyMeta.hd_seed_id.IsNull() && keyMeta.hdKeypath.size() > 0) {
                 // Get the path from the key origin or from the path string
-                // Not applicable when path is "s" as that indicates a seed
+                // Not applicable when path is "s" or "m" as those indicates a seed
```

Comment:
nit: "indicate"

Reply:
Let's point to https://github.com/bitcoin/bitcoin/commit/c75c351419dac3dfe9578604ea5b2f7599452b4a#diff-b2bb174788c7409b671c46ccc86034bdL1470 here or in the functional test.

Reply:
Fixed typo. Added a reference to the PR.